yt/data_objects/construction_data_containers.py

```diff
@@ -1376,8 +1379,16 @@ def _fill_fields(self, fields):
                     refine_by,
                 )
             if level == 0 and tot != 0:
-                raise RuntimeError
+                runtime_errors_count += 1
             self._update_level_state(ls)
+        if runtime_errors_count:
+            warnings.warn(
+                "Something went wrong during field computation. "
+                "This is likely due to missing ghost-zones support "
+                "in class %s" % self.ds.__class__,
```

Comment:
Should the formatting here be similar to what was done in yt-project/yt#2820?

Reply:
and justice for all 🤘 . That being said, since we're actively decluttering the queue on Appveyor right now, I'll wait until flynt PRs are merged to fix this. Thanks for catching it !